composer.json

```diff
@@ -10,5 +10,10 @@
     "require": {
         "php": ">=5.5",
         "zendframework/zendframework": "~2.5"
+    },
+    "autoload": {
+        "psr-4": {
+          "Application\\": "module/Application/src/Application"
```

Comment:
The module is set up as PSR-0 currently.  Could we drop the `Application` folder inside `src` to keep with the intent of PSR-4, or if that's not on then change this to PSR-0? 

And, yes, that's probably the most pedantic programming-related statement I've ever written.